framework/src/st_things/things_stack/cloud/cloud_connector.h

```diff
@@ -93,5 +93,5 @@ extern OCStackResult things_cloud_topic_publish_topic(const char *host,
 		OCRepPayload *payload,
 		OCClientResponseHandler response);
 
-#endif	/* 
+#endif	/*

```

Comment:
I think it's better to erase this comment. (/* */)

Reply:
Thanks, I have fiexd.